packages/odyssey-design-tokens/src/layout.json

```diff
@@ -0,0 +1,5 @@
+{
+  "layout": {
```

Comment:
@arnoldsandoval-okta I'd love your thoughts on where this should live and how it should be named.

Reply:
The max prefix here seems dubious because we go ahead and increase it by another 50% below in the media query. I'd probably just put this under the `font/` directory as a `lineLength.json` file structured like this:

```
{
  "font": {
    "lineLength": {
      "base": { "32rem" }
    }
  }
}
```

Reply:
For clarity, the length here is specific to the type line, not the UI width. That's why it's often modified to include padding or others. 

Your rec makes that much more clear, though! Relocating to `font.lineLength.max` to match our setup for `font.lineHeight.base`